pkg/chains/chains.go

```diff
@@ -113,6 +113,18 @@ var (
 		Name:        "solana_mainnet",
```

Comment:
We need to add the helper functions for TON , such as `IsTonChain` etc .
It might be out of scope for this pr , but please add create an issue and link it in chain.go